tsconfig.json

```
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "target": "ES2022",
+    "module": "nodenext",
+    "moduleResolution": "nodenext",
+    "allowJs": true,
+    "checkJs": true,
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": false,
+    "noEmit": true,
+    "resolveJsonModule": true
+  },
+  "include": ["eslint.config.mjs"],
+  "exclude": ["node_modules", "apps", "packages"]
+}
```

root의 tsconfig는 하위 디렉토리를 모두 제외하고 있는데, 이 경우 해당 설정 파일은 어떤 이유로 작성된 건지 궁금합니다. 오직 eslint.config.mjs를 위한 ts 설정인가요? 저의 경우 해당 파일 삭제하더라도 eslint.config.mjs에 별다른 에러가 발생하지는 않네요...

그외 아래와 같이 수정하는 방식으로 ts 컴파일 최적화가 가능하다는 것 같네요.

```suggestion
{
  "files": [],
  "references": [
    { "path": "./packages/shared" },
    { "path": "./apps/web" },
    { "path": "./apps/api" }
  ]
}
```

또, 하위 tsconfig에서 paths 속성을 삭제하고 composite 속성을  추가해줘야하는 것 같습니다.

---

> 빌드는 기존 Vite 설정을 유지하고, 타입 체크 목적만 분리한 느낌

컴파일 설정도 깎고 계셨군요.....🫠

아따... 마 신기하네요 👍